lib/model.js

```diff
@@ -1044,9 +1044,7 @@ module.exports = (function() {
       options = { fields: options };
     }
 
-    options = Utils._.extend({
-      transaction: null
```

Comment:
I still need to go through the code to find more snippets like this. Passing transaction: null now disables the CLS transaction, so it shouldnt be done anywhere in sequelize code